client/views/clients/clientForm.html

```diff
@@ -4,35 +4,35 @@
 	<div class="form-group">
 		<label class="col-lg-3 control-label">First name:</label>
 		<div class="col-lg-8">
```

Comment:
@PoojaGandhi123 Remove these `required` flags. We don't need mandatory fields on local save. We need to validate these on HMIS save.